### DISCUSSION TITLE
Automatically delete S3 bucket cache objects

### DISCUSSION THREAD

**Discussion by MahdiBM** [runs-on/runs-on#117] [Ideas]
Automatically delete (unused) S3 bucket cache objects after a certain amount of time, like a month.
Will make sure the bucket doesn't incur unnecessary costs, specially after long periods of time.

After a quick search, it appears aws supports "expire"-ing objects after some time. That could also be enough to have. ([`aws s3api put-object --expires`](https://awscli.amazonaws.com/v2/documentation/api/latest/reference/s3api/put-object.html))

**Comment by crohr** [comment]
Hello, this is already the case. Cache objects are expired after 10 days.

**Reply by MahdiBM** [reply]
You're right, I .... thought i noticed they are not deleted, a few weeks ago.
I see there is an expiration set to 10 days.
